plugins/vercel/api_token.go

```diff
@@ -0,0 +1,87 @@
+package vercel
+
+import (
+	"context"
+
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func APIToken() schema.CredentialType {
+	return schema.CredentialType{
+		Name:          credname.APIToken,
+		DocsURL:       sdk.URL("https://vercel.com/docs/cli/login"),
```

Comment:
I think this page has no mention of programmatic authentication (using the API token). Do you have any knowledge of a different documentation page about that?

Reply:
How about this https://vercel.com/docs/rest-api ?